src/redshift-gtk/statusicon.py

```diff
@@ -29,9 +29,13 @@
 import re
 import gettext
 
+import gi
+gi.require_version('Gtk', '3.0')
+
 from gi.repository import Gtk, GLib, GObject
 
 try:
+    gi.require_version('AppIndicator3', '0.1')
```

Comment:
You are right @jonls, I forgot to set the fallback if the required version is not available. I wrote a second patch for that and send it to @genodeftest branch.

(Also, I found that the GtkStatusIcon fallback is not working in my system, I'm 	
looking into this right now)